### PR TITLE
ECIP-1070: ProgPoW for ETC

### PR DESCRIPTION
**Pull request by YazzyYaz**

### CODE REVIEW

File: _specs/ecip-1070.md [hunk 1]
```diff
@@ -0,0 +1,505 @@
+---
+eip: 1057
```
Comment (realcodywburns) [inline _specs/ecip-1070.md]: Should be changed to ref 1070

**Comment by TokenHash** [comment]
🥴🤢🤮

File: _specs/ecip-1070.md [hunk 2]
```diff
@@ -0,0 +1,505 @@
...
+title: ProgPoW, a Programmatic Proof-of-Work
+author: Yaz Khoury <yaz@etccooperative.org>
+discussions-to:
```
Comment (realcodywburns) [inline _specs/ecip-1070.md]: https://github.com/ethereumclassic/ECIPs/issues/172

File: _specs/ecip-1070.md (hunk 1 above)
Comment (YazzyYaz) [reply _specs/ecip-1070.md]: right, will fix

File: _specs/ecip-1070.md (hunk 2 above)
Comment (YazzyYaz) [reply _specs/ecip-1070.md]: just added :)

**Comment by YazzyYaz** [comment]
@fubuloubu damn, I almost had 'em!

**Comment by kimisan** [comment]
ha, I don't see ProgPoW is good for ETC.

**Pull request merged**